t/routes-application.t

```diff
@@ -42,6 +41,8 @@ my $fake-store = mocked(Agrammon::Web::Service,
         get-output-variables => -> $user, $dataset-name {
             %( :variable('x'), :value(2) )
         },
+        get-pdf-export => -> $user, %params {
+        },
```

Comment:
If you're not adding any custom result or behavior to the method, and only want to record its being called, you needn't specify an override for it in the mock.

Reply:
Ah, didn't know. Removed for the moment.